lib/fs.js

```diff
@@ -1209,7 +1209,7 @@ function writeFileSync(path, data, options) {
   }
   let offset = 0;
   let length = data.byteLength;
-  let position = /a/.test(flag) ? null : 0;
```

Comment:
For clarity I would use braces:

```suggestion
  let position = (/a/.test(flag) || isUserFd) ? null : 0;
```